src/main/sensors/rangefinder.h

```diff
@@ -35,6 +35,7 @@ typedef enum {
     RANGEFINDER_MTF02       = 5,
     RANGEFINDER_MTF01P      = 6,
     RANGEFINDER_MTF02P      = 7,
+	RANGEFINDER_TFNOVA      = 8,
```

Comment:
- Replace tab with spaces
```suggestion
    RANGEFINDER_TFNOVA      = 8,
```

Reply:
ok